packages/theme-patternfly-org/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -165,47 +111,77 @@ export const SideNavLayout = ({
               }} />
           </Form>
         </PageHeaderToolsItem>
-      </PageHeaderTools>
-    )
-    : undefined;
+      )}
+    </PageHeaderTools>
+  );
+}
+
+export const SideNavLayout = ({
+  children,
+  location,
+  hideSideNav = false,
+  hasGdprBanner = false,
+  hasFooter = false,
+  hasSearch = false,
+  hasVersionSwitcher = false,
+  sideNavItems,
+  idPages,
+  topNavItems = [],
+  prnum = null,
+  prurl
+}) => {
+  const [versions, setVersions] = useState({ ...staticVersions });
+
+  useEffect(() => {
+    if (typeof window === 'undefined') {
+      return;
+    }
+    if (hasSearch && window.docsearch) {
+      window.docsearch({
+        apiKey: '06941733239da4f8617d272cf2ed4d5c',
+        indexName: 'patternfly',
+        inputSelector: '#global-search-input',
+        debug: false // Set debug to true if you want to inspect the dropdown
+      });
+    }
+    if (hasVersionSwitcher && window.fetch) {
+      fetch('/versions.json')
+        .then(d => d.json())
+        .then(json => setVersions(json))
+        .catch(); // No big deal for core/react
+    }
+  }, []);
+
+  const SideBar = !hideSideNav && (
+    <PageSidebar
+      className="ws-page-sidebar"
+      theme="light"
+      nav={<SideNav location={location} sideNavItems={sideNavItems} idPages={idPages} />}
+    />
+  );
 
-  let headerTitle = title;
-  if (pageSource === 'org') {
-    headerTitle = <Brand src={logo} alt="Patternfly Logo" />;
-  } else if (num) {
-    headerTitle = `PR #${num}`;
-  }
-  
   const Header = (
     <PageHeader
       className="ws-page-header"
-      headerTools={PageTools}
-      logo={headerTitle}
-      logoProps={{
-        href: url || '/'
-      }}
+      headerTools={(hasSearch || hasVersionSwitcher) && getHeaderTools(versions, hasSearch, hasVersionSwitcher)}
+      logo={prnum ? `PR #${prnum}` : <Brand src={logo} alt="Patternfly Logo" />}
+      logoProps={{ href: prurl || '/' }}
```

Comment:
Does it need to account for `v4` in the URL?

Reply:
We redirect https://www.patternfly.org/ to https://patternfly.org/v4 so it's not an issue. To save the user a redirect I'll handle the production build case.